src/frontend/src/pages/new-challenge/new-challenge.tsx

```diff
@@ -1,7 +1,121 @@
-import React from 'react';
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import Button from "@material-ui/core/Button";
+import {
+    Card,
+    CardContent,
+    Typography,
+    CardActions,
+    Container,
+    TextField
+} from "@material-ui/core";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import SubmitChallengePopup from "./components/submit-popup";
 
 export const NewChallenge: React.FC = () => {
     return (
-        <div>New Challenge Page</div>
-    )
+        <div style={{ padding: 50 }}>
+            <Container>
```

Comment:
```suggestion
        <Container>
            <Box p={3}>
```

I would recommend against using inline CSS styles where possible. You can add padding by using the <Box> component with the p(adding) or m(argin) attribute as per https://material-ui.com/system/spacing/